add-ons/ansible-service-broker/ansible-service-broker.addon

```diff
@@ -5,8 +5,9 @@
 # Create ansible-service-broker project
 oc new-project ansible-service-broker
 
-# Add template to Project
-oc create -n ansible-service-broker -f templates/deploy-ansible-service-broker.template.yaml
+# Add upstream broker deployment template to the project
+oc create -n ansible-service-broker -f https://raw.githubusercontent.com/openshift/ansible-service-broker/master/templates/deploy-ansible-service-broker.template.yaml
```

Comment:
Just a note, the template on the master branch will only work with origin versions `>= 3.7` (because of `Cluster*` from the Service Catalog).

Reply:
That's correct, thanks @djzager. It would actually be possible to parametrize that resource name if pre 3.7 support is desired.